lib/matplotlib/backend_managers.py

```diff
@@ -235,16 +228,16 @@ def remove_tool(self, name):
 
     def add_tool(self, name, tool, *args, **kwargs):
         """
-        Add *tool* to `ToolManager`
+        Add *tool* to `ToolManager`.
 
-        If successful adds a new event `tool_trigger_name` where **name** is
-        the **name** of the tool, this event is fired everytime
-        the tool is triggered.
+        If successful, adds a new event ``tool_trigger_{name}`` where ``name``
```

Comment:
```suggestion
        If successful, adds a new event ``tool_trigger_{name}`` where ``{name}``
```
Alternatively, use `[name]` in both places. While curly braces are used for formatting placeholders in python. Square brackets may be more generally common as a placeholder.